styles/widgets/common/popup.less

```diff
@@ -41,3 +41,20 @@
 .dx-dialog-message {
     padding: 10px 10px 5px 10px;
 }
+
+.dx-popup-auto-resizable {
+    &.dx-overlay-wrapper {
+        display: flex;
+        flex-direction: row;
+    }
+
+    .dx-overlay-content {
```

Comment:
Does the fix work without `.dx-overlay-content` css?

Reply:
@jsdmitry Why?

Reply:
```
display: flex;
flex-direction: row;
```
Everything this has inside the flex-container mixin which supports flex styles with other prefixes (browsers compatibility)

Reply:
@jsdmitry We already have [autoprefixer](https://github.com/DevExpress/DevExtreme/pull/5148) for browsers compatibility